src/Microsoft.Framework.Runtime/NuGet/Utility/VersionUtility.cs

```diff
@@ -1006,24 +1023,24 @@ private static Version GetEffectiveFrameworkVersion(FrameworkName projectFramewo
         /// </remarks>
         internal static int GetCompatibilityBetweenPortableLibraryAndPortableLibrary(FrameworkName frameworkName, FrameworkName targetFrameworkName)
         {
-            // Algorithms: Give a score from 0 to N indicating how close *in version* each package platform is the projectâ€™s platforms 
+            // Algorithms: Give a score from 0 to N indicating how close *in version* each package platform is the projectâ€™s platforms
```

Comment:
Either way, it should be cleaned up. I'll do that. Probably a smart quote getting funky.